internal/pkg/otel/manager/manager.go

```diff
@@ -188,6 +189,12 @@ func (m *OTelManager) startCollector(cfg *confmap.Conf, errCh chan error) (conte
 	ctx, cancel := context.WithCancel(context.Background())
 	ap := agentprovider.NewProvider(cfg)
 
+	// enable otel internal logs to include component identifying fields
+	err := featuregate.GlobalRegistry().Set("telemetry.newPipelineTelemetry", true)
```

Comment:
I see breaking changes for this feature gate listed in the release notes for OTel v0.125.0, might be worth a read to see if it affects this in any way: https://github.com/open-telemetry/opentelemetry-collector/releases/tag/v0.125.0